myshopmatic.com.website.json

```diff
@@ -1,21 +1,21 @@
 {  
-   "providerId":"goshopmatic.hk",
+   "providerId":"goshopmatic.com",
```

Comment:
The file should be called goshopmatic.com.website.json with this providerId

Reply:
Good catch - updated file name per suggestion.